[2025-09-03T02:22:12.578+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: institution_detail_data_quality_pipeline.valid manual__2025-09-03T02:09:52.310750+00:00 [queued]>
[2025-09-03T02:22:12.586+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: institution_detail_data_quality_pipeline.valid manual__2025-09-03T02:09:52.310750+00:00 [queued]>
[2025-09-03T02:22:12.586+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T02:22:12.586+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-09-03T02:22:12.586+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T02:22:12.595+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): valid> on 2025-09-03 02:09:52.310750+00:00
[2025-09-03T02:22:12.599+0000] {standard_task_runner.py:55} INFO - Started process 2015 to run task
[2025-09-03T02:22:12.601+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'institution_detail_data_quality_pipeline', 'valid', 'manual__2025-09-03T02:09:52.310750+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/dag_dataquality.py', '--cfg-path', '/tmp/tmpjgnymcp1']
[2025-09-03T02:22:12.602+0000] {standard_task_runner.py:83} INFO - Job 45: Subtask valid
[2025-09-03T02:22:12.614+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-03T02:22:12.657+0000] {task_command.py:388} INFO - Running <TaskInstance: institution_detail_data_quality_pipeline.valid manual__2025-09-03T02:09:52.310750+00:00 [running]> on host 72c28e8237fc
[2025-09-03T02:22:12.711+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Klaus_Rezende
AIRFLOW_CTX_DAG_ID=institution_detail_data_quality_pipeline
AIRFLOW_CTX_TASK_ID=valid
AIRFLOW_CTX_EXECUTION_DATE=2025-09-03T02:09:52.310750+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-09-03T02:09:52.310750+00:00
[2025-09-03T02:22:12.712+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-09-03T02:22:12.713+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo '✅ Data Quality validation passed'"]
[2025-09-03T02:22:12.721+0000] {subprocess.py:86} INFO - Output:
[2025-09-03T02:22:12.724+0000] {subprocess.py:93} INFO - ✅ Data Quality validation passed
[2025-09-03T02:22:12.724+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-03T02:22:12.744+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=institution_detail_data_quality_pipeline, task_id=valid, execution_date=20250903T020952, start_date=20250903T022212, end_date=20250903T022212
[2025-09-03T02:22:12.773+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-09-03T02:22:12.785+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
